src/notation/view/notationviewinputcontroller.h

```diff
@@ -151,8 +151,8 @@ class NotationViewInputController : public muse::actions::Actionable, public mus
     void doZoomToWholePage();
     void zoomToTwoPages();
     void doZoomToTwoPages();
-    void moveScreen(int direction);
-    void movePage(int direction);
+    void moveScreen(int direction, bool userTriggeredMove = true);
+    void movePage(int direction, bool userTriggeredMove = true);
```

Comment:
Here too, I would suggest not adding a `userTriggeredMove` parameter until it's actually needed.